### PR TITLE
Frontend Styling Changes to Breadcrumbs and Sitemap

### PR DESCRIPTION
**Pull request by GameDog9988**
- Upgraded dependencies (probably not necessary).
- Added "items-center" to breadcrumbs content to align vertically.
- Created a `CardView` component for the Sitemap list that matches the look of the `GuideListing` component on the guides page.

I tried upgrading Tailwind to the latest version (would rather have mt-0.5 rather than mt-1 for the arrowSvg) but was receiving an error, curious to see if anyone else encounters it.

### CODE REVIEW

**Review by varughese — APPROVED** [review]
Can you post a screen shot?

**Comment by GameDog9988** [comment]
image
- So that's when you click on one of the types of guides to view, a card grid instead of unordered list tree (which was sick how you did by the way, mapping over the children and calling the tree component again).
- Matched the styling to the cards we had previously in case we wanted to not change it.
image
- The tree view is still on the main guide page for the full guide listing.

**Comment by GameDog9988** [comment]
Something else I wanted to mention was the gitAuthorTime was not working for me, got an error like:
`fatal: invalid object name 'C'.` which I think relates to my C drive.

**Comment by Zmwang622** [comment]
I think the cards might be a bit too tall? It feels like a lot of space gets wasted since now guide title is that long

File: src/components/breadcrumb.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ export default ({ slug, firstBreadCrumb = { link: "/", text: "HOME" } }) => {
   })
 
   return (
-    <div className="breadcrumbs flex md:flex-row flex-col md:items-center mb-3">
-      <span>
+    <div className="breadcrumbs flex flex-wrap items-center mb-3">
+      <span className="flex items-center">
```
Comment (varughese) [inline src/components/breadcrumb.js:47]: The other thing to check is that this still looks good on mobile

image

**Comment by varughese** [comment]
for the git author time, yeah could be related to this not working on windows https://github.com/PittCSWiki/pittcswiki/blob/ef03b24dd6d58b5e0bce70ead86028d7658fbfff/gatsby-node.js#L30

File: src/components/breadcrumb.js (hunk 1 above)
Comment (GameDog9988) [reply src/components/breadcrumb.js:47]: So here's what it looks like on mobile, I think it going in a row makes more sense and flex-wrap is reliable as far as pushing any that overflow to the next line.
image

**Comment by GameDog9988** [comment]
Here's the new height for the cards.
image

File: src/components/breadcrumb.js (hunk 1 above)
Comment (varughese) [reply src/components/breadcrumb.js:47]: I like the vertical more, but agree in situations where there are a lot of breadcrumbs it does get a little too much, so sounds good

**Review by varughese — APPROVED** [review]

**Review by EParmar18 — APPROVED** [review]
looks good

**Pull request merged**